test/project_types/node/commands/connect_test.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+require 'project_types/node/test_helper'
+
+module Node
+  module Commands
+    class CreateTest < MiniTest::Test
```

Comment:
👋 just browsing because this is neat. This is very small but I noticed that this class should probably be `ConnectTest` instead of `CreateTest`. Same with `test/project_types/rails/commands/connect_test.rb` 🙂